app/services/invitations/save_and_send.rb

```diff
@@ -40,9 +40,15 @@ def verify_creneaux_are_available
       return if retrieve_creneau_availability.creneau_availability
 
       fail!(
-        "L'envoi d'une invitation est impossible car il n'y a plus de créneaux disponibles. " \
-        "Nous invitons donc à créer de nouvelles plages d'ouverture depuis l'interface " \
-        "RDV-Solidarités pour pouvoir à nouveau envoyer des invitations"
+        "<strong>Il n'y a plus de créneaux disponibles</strong> pour inviter cet utilisateur. " \
+        "<br/><br/>" \
+        "Nous invitons à créer de nouvelles plages d'ouverture depuis l'interface " \
+        "RDV-Solidarités pour pouvoir à nouveau envoyer des invitations." \
```

Comment:
La formulation de départ est un peu trompeuse parce que la plupart du temps il faut augmenter le délai de prise de rdv plutôt que de créer une plage d'ouverture. 
Peut-être qu'on peut le dire si c'est pas trop lourd 
```suggestion
        "Nous invitons à créer de nouvelles plages d'ouverture ou augmenter le délai de prise de rdv " \
        "depuis l'interface RDV-Solidarités pour pouvoir à nouveau envoyer des invitations." \
```

Reply:
j'ai fait le changement de wording, comme il n'y a eu que ça comme changement je me permet de merge dans la foulée